OpenRA.Game/InstalledMods.cs

```diff
@@ -72,23 +72,27 @@ Manifest LoadMod(string id, string path)
 					throw new InvalidDataException(path + " is not a valid mod package");
 
 				package = new Folder(path);
-				if (!package.Contains("mod.yaml"))
-					throw new InvalidDataException(path + " is not a valid mod package");
```

Comment:
I'll add missing icon.png into log. What about modcontent and all? Can we add some dummy icon.png? Otherwise either I add an exception for these mods in code or it woud be written on each launch, what this PR aims to solve for another error.